modules/benchmark/R/metric.R2.R

```diff
@@ -1,10 +1,12 @@
 ##' @name metric.R2
-##' @title Coefficient of Determination
+##' @title Coefficient of Determination (R2)
 ##' @export
 ##' @param dat dataframe
 ##' 
 ##' @author Betsy Cowdery
+
 metric.R2 <- function(dat, ...) {
+  logger.info("Metric: Coefficient of Determination (R2)")
```

Comment:
@serbinsh since we can't use superscript you want to use subscript?  I think R2 is fine, or alternatively R^2